### PR TITLE
tf-lite CMake: add headers into sources to correctly install them

### PR DESCRIPTION
**Pull request by lbertho-gpsw**
The headers `delegate.h` and `delegate_options.h` were missing when tensorflow lite was installed with the GPU delegates using CMake.

This PR just add these 2 files into  `TFLITE_DELEGATES_GPU_SRCS` variable and it fixes the issue. It works because `TFLITE_DELEGATES_GPU_SRCS` is used to create `_ALL_TFLITE_SRCS` which is used to have `_ALL_TFLITE_HDRS` and then all headers from this variables are installed.

### CODE REVIEW

**Review by qukhan — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @lbertho-gpsw Can you please sign CLA? Thank you!

**Comment by lbertho-gpsw** [comment]
> Hi @lbertho-gpsw Can you please sign CLA? Thank you!

I'm still waiting for the approval of the legal team of my company (it should not be a problem, but it can take some time...) but as soon as I have it I will sign it.

**Comment by gbaned** [comment]
Hi @lbertho-gpsw I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen! Thank you for your contribution.

**Comment by lbertho-gpsw** [comment]
The CLA is signed now, but apparently I can't reopen the PR, should I create a new one ?

**Comment by gbaned** [comment]
> The CLA is signed now, but apparently I can't reopen the PR, should I create a new one ?

Hi @lbertho-gpsw Sure, please create a new one. Thank you!

**Pull request closed without merging**
